processing/src/main/java/io/druid/query/QueryInterruptedException.java

```diff
@@ -109,6 +110,18 @@ public String getHost()
     return host;
   }
 
+  @Override
+  public String toString()
+  {
+    return StringUtils.nonStrictFormat(
```

Comment:
nonStrictFormat() is designed mostly to be called indirectly, e. g. in a method or constructor which accepts `Object... args` itself. In this case I see no point to use it instead of `StringUtils.format()`.